src/autogluon_dashboard/dashboard.py

```diff
@@ -2,23 +2,64 @@
 import subprocess
 import sys
 
-if __name__ == "__main__":
-    csv1 = sys.argv[2]
-    csv2 = sys.argv[4]
+import boto3
+import botocore
+
+BUCKET = "dashboard-test-yash"
+s3_url = "https://dashboard-test-yash.s3.us-west-2.amazonaws.com/"
+
+
+def upload_to_s3(s3_client: botocore.client, file_name: str, object_name: str):
+    """
+    Uploads a file from local filesystem to a specified S3 bucket
+
+    Parameters
+    ----------
+    s3_client: botocore.client.S3,
+        S3 client to interact with bucket
+    file_name: str,
+        Name of local file to upload to S3
+    object_name: str,
+        Name of object to store file contents in S3 bucket
+    """
+    global BUCKET
+    with open(file_name, "rb") as f:
+        s3_client.upload_fileobj(f, BUCKET, object_name)
```

Comment:
Can you check if this upload to subfolders of a bucket? i.e. s3://autogluon-dashboard/PR-XXX/COMMIT/
The `PR-XXX/COMMIT/` is called prefix in s3. I'm not sure if `upload_fileobj` can do that

Reply:
Will take a look and update